src/config/email.ts

```diff
@@ -0,0 +1,13 @@
+import nodemailer from 'nodemailer'
+
+const transporter = nodemailer.createTransport({
+  host: process.env.EMAIL_HOST,
+  port: parseInt(process.env.EMAIL_PORT),
+  secure: false, // upgrade later with STARTTLS
+  auth: {
+    user: process.env.EMAIL_USER,
+    pass: process.env.EMAIL_PWD
```

Comment:
These env vars should be added to `.env.example` with some dummy data to indicate there too that we use them.